airbyte-integrations/connectors/source-surveycto/source_surveycto/helpers.py

```diff
@@ -13,6 +13,20 @@
 
 
 class Helpers(object):
+
+    @staticmethod
+    def modify_date(response_json):
+        data_mod = ["starttime", "endtime", "CompletionDate", "SubmissionDate"]
+        dateformat_in = "%b %d, %Y %I:%M:%S %p"
+        dateformat_out = "%Y-%m-%dT%H:%M:%S+00:00"
+
+        for data in response_json:
+            for key in data_mod:
+                if key in data:
+                    data[key] = datetime.strptime(data[key], dateformat_in).strftime(dateformat_out)
+
+                yield data
```

Comment:
You're creating a function to use only one time. Also now you're doing a double for loop in your data besides only one. I don't think it is the best strategy here.